File: resources/expected/ros2/datatype-examples/strict/src/datatypes_system_cpp_pkg/src/base_code/Sys_i_Instance_consumer_consumer_runner.cpp
#include "datatypes_system_cpp_pkg/user_headers/Sys_i_Instance_consumer_consumer_src.hpp"

//=================================================
//  D O   N O T   E D I T   T H I S   F I L E
//=================================================

Sys_i_Instance_consumer_consumer::Sys_i_Instance_consumer_consumer() : Sys_i_Instance_consumer_consumer_base()
{
    // Invoke initialize entry point
    initialize();

    PRINT_INFO("Sys_i_Instance_consumer_consumer infrastructure set up");
}

int main(int argc, char **argv)
{
    rclcpp::init(argc, argv);
    auto executor = rclcpp::executors::MultiThreadedExecutor();
    auto node = std::make_shared<Sys_i_Instance_consumer_consumer>();
    executor.add_node(node);
    executor.spin();
    rclcpp::shutdown();
    return 0;
}

File: resources/expected/ros2/python-datatype-examples/strict/src/datatypes_system_py_pkg/datatypes_system_py_pkg/base_code/Sys_i_Instance_consumer_consumer_runner.py
#!/usr/bin/env python3
import rclpy
from rclpy.node import Node
from datatypes_system_py_pkg.user_code.Sys_i_Instance_consumer_consumer_src import Sys_i_Instance_consumer_consumer
#=================================================
#  D O   N O T   E D I T   T H I S   F I L E
#=================================================

class Sys_i_Instance_consumer_consumer(Sys_i_Instance_consumer_consumer_base):
  def __init__(self):
    # invoke initialize entry point
    super().__init__()

    Sys_i_Instance_consumer_consumer()

    self.get_logger().info("Sys_i_Instance_consumer_consumer infrastructure set up")

def main(args=None):
  rclpy.init(args=args)
  node =  Sys_i_Instance_consumer_consumer()
  executor = MultiThreadedExecutor()
  executor.add_node(node)
  executor.spin()
  rclpy.shutdown()

if __name__ == "__main__":
  main()
